vue-social/src/router/index.js

```diff
@@ -7,6 +7,7 @@ import UserList from '../views/ListUsers.vue'
 import Chat from '../components/Chat.vue'
 import Room from '../components/Room.vue'
 import AddRoom from '../components/AddRoom.vue'
+import listFriends from "../views/ListFriends";
```

Comment:
change listFriends to ListFriends, follow the code convention

Reply:
changed it.